include/Gaffer/Context.h

```diff
@@ -254,6 +254,15 @@ class GAFFER_API Context : public IECore::RefCounted
 
 				const Context *context() const { return m_context.get(); }
 
+				// Forward compatibility with Gaffer 0.60
+				template<typename T, typename = typename std::enable_if< !std::is_base_of<IECore::Data,T>::value >::type >
```

Comment:
I'm not a big fan of having `enable_if` in the public API, as it's very hard to read. At a minimum I think it should be `typename Enabler = ` so we're communicating what it is for. Then use `typename Enabler` in the definition as well so there's a clear match.